packages/app/components/drop/create-drop-steps/create-drop-steps.tsx

```diff
@@ -0,0 +1,1179 @@
+import { useState, useRef, useEffect, useMemo, useCallback } from "react";
```

Comment:
keeping every step in this file. We can split into different but i think it's fine. We need some pieces for paid drops. I'll restructure once we start working on paid drops.